<ui version="4.0" stdsetdef="1" >
  <author>Sebastian Trueg</author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>base_K3bThemeOptionTab</class>
  <widget class="QWidget" name="base_K3bThemeOptionTab" >
    <layout class="QVBoxLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <item>
        <widget class="QGroupBox" name="groupBox4" >
          <property name="title" >
            <string>Theme Selection</string>
          </property>
          <layout class="QVBoxLayout" >
            <item>
              <widget class="K3ListView" name="m_viewTheme" >
                <property name="allColumnsShowFocus" >
                  <bool>true</bool>
                </property>
                <column>
                  <property name="text" >
                    <string>Theme</string>
                  </property>
                  <property name="clickable" >
                    <bool>true</bool>
                  </property>
                  <property name="resizable" >
                    <bool>true</bool>
                  </property>
                </column>
                <column>
                  <property name="text" >
                    <string>Author</string>
                  </property>
                  <property name="clickable" >
                    <bool>true</bool>
                  </property>
                  <property name="resizable" >
                    <bool>true</bool>
                  </property>
                </column>
                <column>
                  <property name="text" >
                    <string>Version</string>
                  </property>
                  <property name="clickable" >
                    <bool>true</bool>
                  </property>
                  <property name="resizable" >
                    <bool>true</bool>
                  </property>
                </column>
                <column>
                  <property name="text" >
                    <string>Comment</string>
                  </property>
                  <property name="clickable" >
                    <bool>true</bool>
                  </property>
                  <property name="resizable" >
                    <bool>true</bool>
                  </property>
                </column>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="groupBox5" >
          <property name="title" >
            <string>Preview</string>
          </property>
          <layout class="QHBoxLayout" >
            <item>
              <layout class="QHBoxLayout" >
                <property name="spacing" >
                  <number>0</number>
                </property>
                <item>
                  <widget class="QLabel" name="m_leftPreviewLabel" >
                    <property name="text" >
                      <string/>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLabel" name="m_centerPreviewLabel" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>5</hsizetype>
                        <vsizetype>5</vsizetype>
                        <horstretch>1</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="font" >
                      <font>
                        <bold>true</bold>
                      </font>
                    </property>
                    <property name="text" >
                      <string>No theme selected</string>
                    </property>
                    <property name="alignment" >
                      <set>Qt::AlignCenter</set>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLabel" name="m_rightPreviewLabel" >
                    <property name="text" >
                      <string/>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <spacer name="spacer3" >
              <property name="sizeHint" >
                <size>
                  <width>1</width>
                  <height>1</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="m_buttonInstallTheme" >
              <property name="text" >
                <string>Install New Theme...</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="m_buttonRemoveTheme" >
              <property name="text" >
                <string>Remove Theme</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>K3ListView</class>
      <extends>Q3ListView</extends>
      <header location="global" >k3listview.h</header>
    </customwidget>
  </customwidgets>
  <includes>
    <include location="local" >k3listview.h</include>
  </includes>
</ui>
